modules/priors/vignettes/priors_demo.Rmd

```diff
@@ -160,13 +176,14 @@ wullschleger.data <- data.frame(Y  = wullschleger.data$corrected.vcmax,
                                 ft = wullschleger.data$functional.type,
                                 n  = wullschleger.data$neff)
 ## Summarize data by species
-wullschleger.vcmax <- ddply(wullschleger.data, 
-                            .(sp),
-                            summarise, 
-                            Y = mean(Y), 
-                            obs.prec = 1/sqrt(sum((1/obs.prec)^2)), 
-                            ft = mean(ft), # identity
-                            n =  sum(n)) [,-1]
+library(dplyr)
```

Comment:
already loaded dplyr at the top of the file — is this call redundant or is it needed for some  Rmarkdown per-chunk subtlety here?

Reply:
fixed in https://github.com/PecanProject/pecan/pull/2439/files/272fbc199168822d2486425a64e9236b3a697617